config/schemas/config.yaml

```diff
@@ -1515,34 +1515,13 @@ properties:
       excludedExtraNamespaces:
         title: Excluded Extra Namespaces
         description: |-
-          Extra excluded namespace, here we should add the namespaces that are more likely to change
+          Extra namespaces excluded by HNC, here we should add the namespaces that are more likely to change
 
           [Including and excluding namespaces](https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/master/docs/user-guide/how-to.md#including-and-excluding-namespaces-from-hnc)
-        items:
-          type: string
-        type: array
-      excludedNamespaces:
-        title: Excluded Namespaces
-        description: |-
-          Excluded namespaces
```

Comment:
Probably not many are overriding this but since we are removing a configuration option that should probably be mentioned as a breaking change.

Reply:
Btw is it removed from the default config when you re-run `ck8s init`?

Reply:
> Btw is it removed from the default config when you re-run `ck8s init`?

It should, but I can double check

Reply:
Yes it is removed from default config